frontend/src/pages/SearchResultPage/index.tsx

```diff
@@ -1,30 +1,53 @@
-import { useRef, useState } from 'react';
+import { useEffect, useRef } from 'react';
+import { useInfiniteQuery } from 'react-query';
 import { RouteComponentProps } from 'react-router-dom';
+import API from 'src/apis/requests';
 import Arrow from 'src/components/@shared/Arrow/Arrow';
 
 import Header from 'src/components/@shared/Header/Header';
 import InfinityScrollPoll from 'src/components/@shared/InfinityScrollPoll/InfinityScrollPoll';
 import ListItem from 'src/components/Item/ListItem';
 import List from 'src/components/List/List';
 import { PATH } from 'src/constants';
+import useInfinityScroll from 'src/hooks/useInfinityScroll';
+import { categories } from '../SearchPage';
 
 import { Container, Title, Section } from './styles';
 
-const defaultSearchResult = [
-  {
-    id: 0,
-    name: 'name',
-    alcoholByVolume: 0,
-    imageUrl: 'https://fakeimg.pl/88x88',
-  },
-];
-
 const SearchResultPage = ({ history, location }: RouteComponentProps) => {
-  const infinityPollRef = useRef(null);
-
-  const [searchResult, setSearchResult] = useState(defaultSearchResult);
+  const observerTargetRef = useRef<HTMLDivElement>(null);
 
   const words = new URLSearchParams(location.search).get('words');
+  const categoryKey = new URLSearchParams(location.search).get('category');
+
+  const categoryName =
+    categoryKey && categories.find((category) => category.key === categoryKey)?.name;
```

Comment:
여기 사실.. 제가 이부분은 간단하게만 넣었는데
{
      'BEER' : {
          name: '맥주',
          Icon: BeerIcon},
      ....
}

이런식으로 하면 
```suggestion
categoryName = categories[categoryKey]?.name ?? '' 
```
처럼 작성할 수도 있을 것 같아요~

Reply:
맞아 이거 key값 일부러 안뺀건가 했는데, 역시 서니는 생각이 다 있었군요..! 
이 부분은 나중에 카테고리를 관리하는 테이블로 따로 뺄 때 수정해도 괜찮을거 같은데 어떻게 생각하시나용!?!